### ISSUE TITLE
Warnings are not reported on valid documents

### ISSUE THREAD

**Issue by caleb531** [renyard/validity#70]
Hi,

I recently discovered that Validity does not report warnings from the W3C Validator, and therefore request that Validity report warnings in the console (in addition to errors).

## Test case

Here's an HTML page containing some broken table markup (the `td` counts per row do not match):

```html
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8" />
	<title>Warning Page</title>
</head>
<body>

<h1>Warning Page</h1>

<table>
	<tbody>
		<tr>
			<th>Fruit</th>
		</tr>
		<tr>
			<td>Apple</td>
			<td>Banana</td>
		</tr>
	</tbody>
</table>

</body>
</html>
```

When I check this code using Validity, it reports that the markup is completely valid (`Document is valid ("HTML5")`). However, when I run the code through the official W3C Validator, I receive a single warning:

<img width="967" alt="screen shot 2016-09-15 at 1 06 21 pm" src="https://cloud.githubusercontent.com/assets/872474/18565834/3f6dbc4c-7b46-11e6-8933-843d4c27e604.png">

Therefore, I please ask that warnings be reported (by default, or enabled through an extension setting). I need my markup to pass the W3C Validator completely for my project.

Thanks,
Caleb

**Comment by renyard** [comment]
Hi @caleb531

Thanks for reporting. Validity does report warnings as well as errors, but after testing this it seems there's a bug in that it swallows the warnings if no errors are reported.

Should be quite straightforward to fix, so I'll try and get a fix out in the next few days.

Thanks again
Ian

**Comment by renyard** [comment]
Hi @caleb531

I've just release v3.3.0, which now reports warnings for valid documents.

As with all releases, I've initially deployed this to a subset of users. In order to get the fix immediately, you may need to uninstall Validity before installing again from the [Chrome Web Store](https://chrome.google.com/webstore/detail/bbicmjjbohdfglopkidebfccilipgeif).

Thanks again for reporting this. 👍
